packages/kbn-securitysolution-autocomplete/src/field_value_lists/index.tsx

```diff
@@ -121,7 +121,11 @@ export const AutocompleteFieldListsComponent: React.FC<AutocompleteFieldListsPro
       !allowLargeValueLists && (
         <EuiText size="xs">
           {i18n.LISTS_TOOLTIP_INFO}{' '}
-          <EuiLink external target="_blank" href="https://www.elastic.co/">
+          <EuiLink
+            external
+            target="_blank"
+            href="https://www.elastic.co/guide/en/security/master/detections-ui-exceptions.html#manage-value-lists"
```

Comment:
@dplumlee This is the right docs location, but usually when we review docs links they're built with a variable instead of fully hard-coded, something like `${SECURITY_SOLUTION_DOCS}detections-ui-exceptions.html#manage-value-lists`. I think this helps resolve versioning, but I don't know all the details; feel free to disregard if that's not relevant to this use case.

Reply:
I was unaware we had a tool for this now, I went ahead and updated the link to use that approach

Reply:
Docs link looks good, thanks!